docker/candlepin-base/cp-test

```diff
@@ -254,6 +255,15 @@ if [ "$UNITTEST" -gt 0 ]; then
     done
 fi
 
+if [ "$QPID" == "1" ]; then
+    echo "Setting up Qpid"
+    pushd . 
+    cd ../gutterball/bin/qpid
+    ./configure-qpid.sh
```

Comment:
Since the -q is attempting to configure QPID, perhaps we should do a complete clean first, and then a configure.

```bash
# Do a cleanup
./configure-qpid.sh -c
# Reconfigure
./configure-qpid.sh
```